mycpp/gc_mylib.h

```diff
@@ -105,6 +107,9 @@ class CFileLineReader : public LineReader {
   virtual int fileno() {
     return ::fileno(f_);
   }
+  void close() {
+    fclose(f_);
+  }
```

Comment:
should probably add `f_ = nullptr` here...

Reply:
This code will change anyway -- see Zulip's Unix I/O vs. C I/O

https://oilshell.zulipchat.com/#narrow/stream/121539-oil-dev/topic/Unix.20I.2FO.20instead.20of.20C.20I.2FO

I have a good plan for that